sky/engine/core/text/ParagraphBuilder.cpp

```diff
@@ -136,7 +136,7 @@ ParagraphBuilder::~ParagraphBuilder()
 {
 }
 
-void ParagraphBuilder::pushStyle(Int32List& encoded, const String& fontFamily, double fontSize, double letterSpacing, double wordSpacing, double lineHeight)
+void ParagraphBuilder::pushStyle(Int32List& encoded, const char* fontFamily, double fontSize, double letterSpacing, double wordSpacing, double lineHeight)
```

Comment:
Ah, yeah.  That's busted.  I have a fix locally.  Just testing it out a bit.  This argument becomes a std::string, which can handle embedded U+0000.